src/Utils/FileSystem.php

```diff
@@ -131,11 +131,14 @@ public static function read(string $file): string
 
 	/**
 	 * Writes a string to a file.
+	 * @param $mode int|NULL
```

Comment:
Most importantly you switched parameter name with parameter type, the annotation should be `@param int|NULL $mode`